java.lang.NullPointerException

Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.Object java.lang.ref.Reference.get()' on a null object reference
        at com.iterable.iterableapi.IterableActivityMonitor$2.onActivityPaused(IterableActivityMonitor.java:68)
        at android.app.Application.dispatchActivityPaused(Application.java:234)
        at android.app.Activity.onPause(Activity.java:1664)

--- a/iterableapi/src/main/java/com/iterable/iterableapi/IterableActivityMonitor.java
+++ b/iterableapi/src/main/java/com/iterable/iterableapi/IterableActivityMonitor.java
@@ -65,7 +65,7 @@ public class IterableActivityMonitor {
 
         @Override
         public void onActivityPaused(Activity activity) {
-            if (currentActivity.get() == activity) {
+            if (getCurrentActivity() == activity) {
                 currentActivity = null;
             }
         }

